application.py

```diff
@@ -378,6 +378,95 @@ def add_bank_details():
     ]
 }
 
+menus['individual'] = {
+    'intro': "This is the Individual Menu",
+    'options': [
+        {
+            'friendly': 'My Social Grants',
+            'dest': {
+                'type': DEST_TYPE_TASK,
+                'value': 'business_loans'
+            },
+        },
+        {
+            'friendly': 'All Social Grants',
+            'dest': {
+                'type': DEST_TYPE_MENU,
+                'value': 'all_grants'
+            },
+        },
+        {
+            'friendly': 'My Profile',
+            'dest': {
+                'type': DEST_TYPE_TASK,
+                'value': 'profile_menu'
```

Comment:
Not sure, but I guess that should trigger a new menu to be able to:
- get the user profile
- delete the user profile
- going back

@rossguy maybe you had something else in mind here?